cadc-uws-server/src/main/java/ca/nrc/cadc/uws/web/restlet/resources/UWSResource.java

```diff
@@ -91,8 +92,10 @@
 import org.restlet.resource.ServerResource;
 
 import ca.nrc.cadc.auth.AuthenticationUtil;
+import ca.nrc.cadc.net.NetUtil;
```

Comment:
Parameter and NetUtil are imported here because of a indirect reference through RestletWebUtil?